src/all/holonometria/src/eu/kanade/tachiyomi/extension/all/holonometria/Holonometria.kt

```diff
@@ -22,31 +21,23 @@ class Holonometria(
 
     override val name = "HOLONOMETRIA"
 
-    override val baseUrl = "https://alt.hololive.tv"
+    override val baseUrl = "https://holoearth.com"
 
     override val supportsLatest = false
 
-    override val client = network.cloudflareClient.newBuilder()
-        .readTimeout(60, TimeUnit.SECONDS)
-        .build()
+    override fun popularMangaRequest(page: Int) = GET("$baseUrl/${langPath}alt/holonometria/manga/")
```

Comment:
```suggestion
    override fun popularMangaRequest(page: Int) = GET("$baseUrl/${langPath}alt/holonometria/manga/", headers)
```
pass headers to `GET` here and elsewhere